### DISCUSSION TITLE
TRBD d_call not matched blastn and CDR3AA differents from RTCR

### DISCUSSION THREAD

**Discussion by pengbo233** [milaboratory/mixcr#1415] [Q&A] [answered]
Hello, thank you very much for providing the RTCR code. It helps with replacing mixcr for analysis.
However, I have noticed two issues:

Q1 I noticed that the nucleotide sequences for CDR3 in RTCR and mixcr are identical, but the corresponding amino acid sequences are different.

Use RTCR to analyze the CDR3 sequences.
TGTGCCAGCAGTTTATGTCGACTAGCGGCACCTACGAGCAGTACTTC CASSLCRLAAPTSST
Use mixcr to analyze the CDR3 sequences.
TGTGCCAGCAGTTTATGTCGACTAGCGGCACCTACGAGCAGTACTTC CASSLCRL_GTYEQYF
Compare the nucleotide and amino acid sequences obtained from RTCR and mixcr.
_GTYEQYF  AAPTSST

Q2 I used blastn to compare TRBD short sequence and found that the results were consistent with mixcr's accuracy, but mixcr had a wider coverage. Could you please tell me how to set the parameters of mixcr short sequence, because from the mixcr results, it seems that three bases are the lowest matching? Is this too low? I set 4. Is it feasible?

**Comment by mizraelson** [comment, marked as answer]
Hi,
1) It appears there may be a frame-shift somewhere in the sequence, leading to a stop-codon within the CDR3 region. While I'm not certain, I suspect that RTCR translates the CDR3 region in isolation from the rest of the sequence, potentially overlooking non-functional CDR3s. Do any of the covered regions exhibit a frame-shift?

2) I'm not entirely clear on what you mean by "wider coverage." Could you provide a detailed example or elaborate further on this point? Your additional information will help me respond more accurately.

Sincerely,
Mark

**Comment by mizraelson** [comment]
Will close for now. Feel free to reopen.
